pkg/cvo/sync_worker.go

```diff
@@ -329,7 +329,9 @@ func (w *SyncWorker) Start(ctx context.Context, maxWorkers int) {
 				utilruntime.HandleError(fmt.Errorf("unable to synchronize image (waiting %s): %v", interval, err))
 				continue
 			}
-			klog.V(5).Infof("Sync succeeded, reconciling")
+			if work.State != payload.ReconcilingPayload {
+				klog.V(4).Infof("Sync succeeded, transitioning from %s to %s", work.State, payload.ReconcilingPayload)
```

Comment:
Doesv(4) gets printed in the CVO log by default?

Reply:
yup.  See our existing CVO logs for lovely `Running sync...in state...` entries which I refer to all the time, because we currently don't bubble the failing manifests up into the ClusterVersion.